libavpipe/src/avpipe_io.c

```diff
@@ -208,17 +209,21 @@ elv_io_close(
     ioctx_t *outctx = (ioctx_t *)pb->opaque;
     avpipe_io_handler_t *out_handlers = NULL;
 
-    if (out_tracker != NULL)
+
+    if (out_tracker != NULL) {
         out_handlers = out_tracker->out_handlers;
+    }
 
-    elv_dbg("OUT elv_io_close url=%s, stream_index=%d, seg_index=%d avioctx=%p, avioctx->opaque=%p buf=%p outtracker[0]->last_outctx=%p, outtracker[1]->last_outctx=%p",
+    elv_dbg("OUT elv_io_close url=%s, stream_index=%d, seg_index=%d avioctx=%p, avioctx->opaque=%p buf=%p outtracker->last_outctx=%p, outhandlers=%p",
         outctx != NULL ? outctx->url : "", outctx != NULL ? outctx->stream_index : -1, outctx != NULL ? outctx->seg_index : -1, pb, pb->opaque, avioctx->buffer,
-	    out_tracker != NULL ? out_tracker[0].last_outctx : 0, out_tracker != NULL ? out_tracker[1].last_outctx : 0);
+	    out_tracker != NULL ? out_tracker->last_outctx : 0, out_handlers);
     if (out_handlers) {
-#if 0
-        // PENDING (RM): this crashes with ffmpeg-4.4, needs to be fixed
-        out_handlers->avpipe_stater(outctx, out_stat_encoding_end_pts);
-#endif
+        // TODO(Nate): Separate out this stat into something more descriptive of the particular case
+        // For now, this double-stat is fine because the 'out_stat_encoding_end_pts' is also used
+        // for muxing, which doesn't have a meaningful value of 'seg_index'. Additionally, ABR and
+        // mez should be pretty separate. But that can be done later.
+        out_handlers->avpipe_stater(outctx, out_tracker->output_stream_index, out_stat_encoding_end_pts);
+        out_handlers->avpipe_stater(outctx, out_tracker->output_stream_index, out_stat_end_file);
```

Comment:
No longer crashes?

Reply:
I never saw it crash. Not sure what case it would, but all tests I ran worked fine.